### PR TITLE
Add new production inventory pointing to next

### PR DESCRIPTION
**Pull request by sauloperez**
This new inventory host will replace `next.timeoverflow.org` when the migration is complete.

Now, the three hosts will coexist while we perform the migration which will allow us to run any necessary ansible scripts against them.

:warning: this will also lead to a new certificate created by certbot since `inventory_hostname` will change from `next.timeoverflow.org` to `timeoverflow.org`.

### CODE REVIEW

File: inventory/host_vars/timeoverflow.org/secrets.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  31333138323830663637646264623163326661333866356339323035643936633964383264613064
+  3530383037623666623830666134356534643335336431330a633736323732363039356266393934
+  36633162326136313037313861633536323362363261323566663134663933343134616462663430
+  6362316131356565360a303464333062353131323730323166613639653464316464356236663865
+  66613363626163313361616439663531363832333130613065303738626562343234383538333636
+  3461633966356639613066396639643037393037333434623832
+
+backups_role_postgresql_user_password: "{{ database_password }}"
+
+backups_role_restic_repo_url: !vault |
```
Comment (sauloperez) [inline inventory/host_vars/timeoverflow.org/secrets.yml:45]: we will mix next and production backups but I think that's just fine.

**Review by enricostano — APPROVED** [review]

**Comment by sauloperez** [comment]
Now the only difference between this and the `next.timeoverflow.org` host inventory is just the `rails_environment`.

File: inventory/hosts [hunk 2]
```diff
@@ -7,11 +7,19 @@ staging.timeoverflow.org ansible_host=116.203.113.233
 [next]
 next.timeoverflow.org ansible_host=78.47.225.84
 
+[production]
+timeoverflow.org ansible_host=78.47.225.84
```
Comment (sauloperez) [inline inventory/hosts:11]: Note this is the same machine as `next` above :point_up:

**Review by danypr92 — APPROVED** [review]
Why we separate the `config.yml` and the `secrets.yml`? Now, using the `vault_string` we can put all the vars together or split by theme instead of `config` and `secrets`

**Comment by sauloperez** [comment]
I guess we didn't merge them in this case too keep it easy to compare both inventories. I'll create an issue.

**Pull request merged**